spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/StreamsController.java

```diff
@@ -67,4 +75,27 @@ public StreamsController(StreamDeployer streamDeployer, StreamDefinitionReposito
 	protected StreamDefinition createDefinition(String name, String definition) {
 		return new StreamDefinition(name, definition);
 	}
+
+	@ResponseBody
+	@RequestMapping(value = "/clean/{transport}/{stream}")
+	@ResponseStatus(HttpStatus.OK)
+	public List<String> clean(@PathVariable String transport, @PathVariable String stream,
+			@RequestParam(required = false) String adminUri,
```

Comment:
Given the specific options that are here, I believe we should have a dedicated endpoint (ie `"/clean/rabbit/{stream}"`) as this won't accommodate other transports anyway